provider/resources.go

```diff
@@ -16,22 +16,29 @@ package hcloud
 
 import (
 	"fmt"
+	_ "embed"
 	"path/filepath"
 	"unicode"
 
 	"github.com/hetznercloud/terraform-provider-hcloud/hcloud"
 	"github.com/pulumi/pulumi-hcloud/provider/pkg/version"
 	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge/x"
 	shimv2 "github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfshim/sdk-v2"
 	"github.com/pulumi/pulumi/sdk/v3/go/common/tokens"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/util/contract"
 )
 
+var
```

Comment:
The token position is generated incorrectly from https://github.com/pulumi/upgrade-provider/blob/aqiu/code-migration-ast/migrations/migrations.go#L43-L50